### PR TITLE
Ablation tutorial documentation updated

### PR DESCRIPTION
**Pull request by mkvani**
Hi,

Could you please review this tutorial for documentation?
In addition, there are few additions related to missing steps compared to the original tutorial. 

Thank you,
Manoj

### CODE REVIEW

**Comment by mkundu1** [comment]
The code-style and doc-build CIs are failing. To fix code-style, please setup and run pre-commit as mentioned [here](https://examples.fluent.docs.pyansys.com/version/dev/contributing.html#code-style).

The [documentation failure](https://github.com/ansys/pyfluent-examples/actions/runs/5713821974/job/15479934772?pr=61#step:5:244) is due to missing image files.

File: doc/source/conf.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def _reset_module_cb(gallery_conf, fname, when):
     "backreferences_dir": None,
     # Modules for which function level galleries are created.  In
     "doc_module": "ansys-fluent-core",
-    # "ignore_pattern": r"demo22R2\.py",  # noqa: E501
+    "ignore_pattern": r"00-ahmed_body_workflow\.py|01-brake\.py|01-CreatePPTX\.py|02-DashApp\.py|03-powerPointReport\.py|04-TylerSofrinModes\.py",  # noqa: E501
```
Comment (mkundu1) [inline doc/source/conf.py:152]: This change would not be required

File: examples/01-beta_examples/00-ablation.py [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
-setting up and running the solver, and reviewing the results using Fluent's
-postprocessing capabilities.
+#################################
+Modeling Ablation
+#################################
+
+Objective:
+==========
+
+In this tutorial, a flow field of air passing around a 3D wedge is simulated using the Ansys Fluent ablation model. Ablation is an effective treatment used to protect a vehicle from the damaging effects of external high temperatures. During high speed vehicle operations (for example, vehicle reentry), the ablative coating is chipped away to remove excessive amount of heat from the surface of the vehicle.
```
Comment (seanpearsonuk) [inline examples/01-beta_examples/00-ablation.py:9]: Should the lines be this long? How are they rendered?

Comment (seanpearsonuk) [reply examples/01-beta_examples/00-ablation.py:9]: Regarding "high speed vehicle operations (for example, vehicle reentry)". Really it's the friction against the atomsphere at high speed that's important. High speeds of the same vehicle outside the atmosphere do not have any such effect.  So putting the reentry part just in brackets seems not quite right. Also, "remove excessive amount of heat" is awkward phrasing.

File: examples/01-beta_examples/00-ablation.py [hunk 3]
```diff
@@ -1,18 +1,57 @@
...
+
+* Define boundary conditions for a high-speed flow.
+* Set up the ablation model to model effects of a moving boundary due to ablation.
+* Initiate and solve the transient simulation using the density-based solver.
+
+Problem Description:
+====================
+
+
+The geometry of the 3D wedge considered in this tutorial is shown in following Figure. The air flow passes around a nose of a re-entry vehicle operating under high speed conditions. The inlet air has a temperature of 4500 K, a gauge pressure of 13500 Pa, and a Mach number of 3. The domain is bounded above and below by symmetry planes (displayed in yellow). As the ablative coating chips away, the surface of the wall moves. The moving of the surface is modeled using dynamic meshes. The surface moving rate is estimated by Vielle's empirical law:
```
Comment (seanpearsonuk) [inline examples/01-beta_examples/00-ablation.py:21]: should be "in the following figure"

Comment (seanpearsonuk) [reply examples/01-beta_examples/00-ablation.py:21]: I looked up "Vielle's empirical law". This seems to be a misspelling - it's Vieille's Law, also credited as Saint Robert's law.

File: doc/source/conf.py (hunk 1 above)
Comment (mkvani) [reply doc/source/conf.py]: Reverted the changes

File: examples/01-beta_examples/00-ablation.py (hunk 2 above)
Comment (mkvani) [reply examples/01-beta_examples/00-ablation.py]: As per pre-commit, it is getting adjusted. However it is not showing paragraph in "justify" text alignment (please check "Objective" paragraph).

Other changes to objective description made as per suggestion. Please check.

File: examples/01-beta_examples/00-ablation.py (hunk 3 above)
Comment (mkvani) [reply examples/01-beta_examples/00-ablation.py]: Corrected the spelling.

**Comment by mkvani** [comment]
Hi @seanpearsonuk and @mkundu1 , could you please review again and check the resolved commit? 

Thank you,
Manoj

**Review by mkundu1 — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Pull request merged**